examples/PowerElectronics/Microgrid/Microgrid.cpp

```diff
@@ -342,13 +344,14 @@ int main(int argc, char const* argv[])
 
   std::vector<double>& yfinial = sysmodel->y();
 
-  std::cout << "Final Vector y\n";
-  for (size_t i = 0; i < yfinial.size(); i++)
-  {
-    std::cout << yfinial[i] << "\n";
-  }
+  // // Optional debugging output
```

Comment:
Consider adding a verbosity flag and a conditional to get to this code. Or remove completely.

Reply:
Need to add CLI utilities to GridKit first.